recipe/meta.yaml

```diff
@@ -10,13 +10,15 @@ source:
   sha256: 685ca6f7fedda07434d8ee03c536f4763385671c4509c5bb48beb3055fd236ac
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   build:
     - make
     - {{ compiler('c') }}  # [unix]
     - {{ compiler('m2w64_c') }}  # [win]
+    - m2w64-toolchain  # [win]
+    - {{ stdlib("c") }}
```

Comment:
```suggestion
    - {{ compiler('c') }}        # [unix]
    - {{ stdlib('c') }}          # [unix]
    - {{ compiler('m2w64_c') }}  # [win]
    - {{ stdlib('m2w64_c') }}    # [win]
```

Reply:
You then also have to rerender the feedstock, see docs for [bot commands](https://conda-forge.org/docs/maintainer/infrastructure/#admin-web-services).

@conda-forge-admin, please rerender